### PR TITLE
Rename Evd.evar_universe_context -> ustate

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Comment by gares** [comment]
thanks!

**Review by gares — APPROVED** [review]

**Comment by SkySkimmer** [comment]
vscoq needs https://github.com/coq-community/vscoq/pull/841

**Comment by herbelin** [comment]
A related thing which makes difficult the reading of the code about universes currently is the overloading of the role of the identifier `uctx`. If it were uniformly named e.g. `ustate` when an `UState.t`, that would help I think.

**Comment by SkySkimmer** [comment]
CI green

**Comment by gares** [comment]
@coqbot merge now

File: engine/termops.mli [hunk 1]
```diff
@@ -320,3 +319,6 @@ val print_rel_context : env -> Evd.evar_map -> Pp.t
 val print_env : env -> Evd.evar_map -> Pp.t
 
 end
+
+val pr_evar_universe_context : UState.t -> Pp.t
+[@@deprecated "(8.21) Use [Evd.pr_ustate] instead"]
```
Comment (proux01) [inline engine/termops.mli:324]: I can't find that `pr_ustate` anywhere (currently silenced the deprecation warning in rocq-elpi)

Comment (ppedrot) [reply engine/termops.mli:324]: I surmise this is `UState.pr`.

Comment (proux01) [reply engine/termops.mli:324]: Indeed, thanks, I should just have looked at the definition 🤦

**Pull request merged**